example-apps/internal-knowledge-search/app-ui/src/config/documentsToSearchResultMappings.json

```diff
@@ -1,6 +1,6 @@
 {
-  "search-mongo": {
-    "title": "name",
+  "search-spo": {
+    "title": "Title",
```

Comment:
Probably want to undo this, to leave mongo documented?
The bottom of this file has an object for sharepoint